editor/src/clj/editor/workspace.clj

```diff
@@ -291,6 +290,7 @@ ordinary paths."
         (dialogs/make-info-dialog
           {:title "Unable to Load Plugin"
            :icon :icon/triangle-error
+           :always-on-top true
```

Comment:
Snuck this fix in at the last minute. If there's an error in a plugin while loading the project, make sure the error dialog stays on top after the editor opens.